lib/manager/cargo/extract.ts

```diff
@@ -63,13 +85,50 @@ function extractFromSection(
 
 export function extractPackageFile(
   content: string,
-  fileName: string
+  fileName: string,
+  config?: ExtractConfig
```

Comment:
```suggestion
  config: ExtractConfig
```
Config should never be null or undefined.

Reply:
It is in tests - and in the `ManagerApi` interface:

```typescript
export interface ManagerApi {
  extractPackageFile?(
    content: string,
    packageFile?: string,
    config?: ExtractConfig
  ): Result<PackageFile | null>;
}
```

Reply:
Looks like a mistake in api interface. Will fix that soon.

Reply:
Yeahh actually the tests always pass a config too, they just don't always pass a `packageFile` 🤦  My next commit fixes that.